src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -175,10 +143,6 @@ BenefitPaymentsAndDebt.propTypes = {
       accountNumber: PropTypes.string.isRequired,
     }),
   ),
-  paymentsError: PropTypes.bool,
 };
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps,
-)(BenefitPaymentsAndDebt);
+export default connect()(BenefitPaymentsAndDebt);

```

Comment:
If this isn’t hooked into redux anymore, I would go ahead and remove the `connect` and just return the component

Reply:
fixed